pkg/logger/methods.go

```diff
@@ -0,0 +1,108 @@
+package logger
+
+import (
+	"context"
+	"go.uber.org/zap/zapcore"
+)
+
+func Log() *Logger {
+	return &Logger{logger: baseLogger}
```

Comment:
why do we use locks on other functions, but not here?

Reply:
Because we're only returning the struct, it doesn't involve any action